engine/mission/include/hashtable.h

```diff
@@ -19,9 +19,8 @@
  * You should have received a copy of the GNU General Public License
  * along with Vega Strike. If not, see <https://www.gnu.org/licenses/>.
  */
-
-#ifndef _HASHTABLE_H_
-#define _HASHTABLE_H_
+#ifndef VEGA_STRIKE_ENGINE_MISION_HASHTABLE_H
```

Comment:
Missing an `S` in `MISSION`.

```suggestion
#ifndef VEGA_STRIKE_ENGINE_MISSION_HASHTABLE_H
```

Reply:
@stephengtuggy should be fixed